packages/editor/src/components/post-publish-panel/maybe-post-format-panel.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * External dependencies
+ */
+import { find, get, includes } from 'lodash';
+
+/**
+ * WordPress dependencies.
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { ifCondition, compose } from '@wordpress/compose';
+import { withDispatch, withSelect } from '@wordpress/data';
+import { Button, PanelBody } from '@wordpress/components';
+
+/**
+ * Internal dependencies.
+ */
+import { POST_FORMATS } from '../post-format';
+
+const PostFormatSuggested = ( { suggestedPostFormat, suggestionText, onUpdatePostFormat } ) => (
+	<Button isLink onClick={ () => onUpdatePostFormat( suggestedPostFormat ) }>
+		{ suggestionText }
+	</Button>
+);
+
+const PostFormatPanel = ( { suggestion, onUpdatePostFormat } ) => {
+	const panelBodyTitle = [
+		__( 'Suggestion:' ),
+		(
+			<span className="editor-post-publish-panel__link" key="label">
+				{ __( 'Use a post format' ) }
+			</span>
+		),
+	];
+
+	return (
+		<PanelBody initialOpen={ false } title={ panelBodyTitle } >
+			<p>
+				{ __( 'Your theme uses post formats to highlight different kinds of content like images or videos. Apply one to see this special styling' ) }
```

Comment:
This isn't exactly as suggested by @sarahmonster (it's lacking a period and a comma, as well as using "one" instead of the clearer "a post format"). I've pushed a tweak though 😄

Reply:
I should have used copy-pasta!

Reply:
My persnickety heart appreciates this! 🍝❤️